test-storybooks/portable-stories-kitchen-sink/react/stories/__snapshots__/Button.test.tsx.snap

```diff
@@ -2,6 +2,9 @@
 
 exports[`Renders CSF2Secondary story 1`] = `
 <body>
+  <div
```

Comment:
This is a breaking change and will impact users using portable stories for snapshots. This doesn't seem to make sense outside of Storybook (or docs mode), is there a way to avoid this?

Reply:
I could have it only create this when the first highlight is created. I think that makes sense. Would that be sufficient?